### ISSUE TITLE
Feature request: silent mode

### ISSUE THREAD

**Issue by sir-pinecone** [nafeu/gryt-focus#1]
Before I get into the ask, I want to know if you have any intentions to allow the app to operate as a pure timer instead of a countdown? I tend to work for long uninterrupted spans and would like to simply track the amount of time that passes. Are you using the user set duration in your "focus" calculations? If so then perhaps you can disable features when running in timer mode? 

As for the ask, if you don't want to add a timer mode, then can you add a way to disable the notification bell that goes off when the countdown reaches zero? I didn't realize that was there and it interrupted my focus yesterday haha. It wasn't a big deal, but it made me realize I can't use this unless I set the duration to be really long or I mute my browser :P

**Comment by nafeu** [comment]
@sir-pinecone Yeah I can enable the timer as a stopwatch instead of a countdown timer if you find that to be useful. I can add a mode switch button that disables the countdown as well as the alarm+sound

**Comment by nafeu** [comment]
@sir-pinecone Yeah I can enable the timer as a stopwatch instead of a countdown timer if you find that to be useful. I can add a mode switch button that disables the countdown as well as the alarm+sound
